src/api/XRInputPose.js

```diff
@@ -13,33 +13,39 @@
  * limitations under the License.
  */
 
+import XRRay from './XRRay';
 import * as mat4 from 'gl-matrix/src/gl-matrix/mat4';
 
 export const PRIVATE = Symbol('@@webxr-polyfill/XRInputPose');
 
 export default class XRInputPose {
   /**
    * @param {boolean} emulatedPosition
-   * @param {Float32Array} pointerMatrix
    * @param {Float32Array} gripMatrix
    */
   constructor(inputSourceImpl, hasGripMatrix) {
     this[PRIVATE] = {
       inputSourceImpl,
-      pointerMatrix: mat4.identity(new Float32Array(16)),
-      gripMatrix: hasGripMatrix ? mat4.identity(new Float32Array(16)) : null,
+      targetRay: new XRRay(),
+      gripMatrix: hasGripMatrix ? mat4.create() : null,
     };
   }
 
   /**
-   * @return {boolean}
+   * @return {XRRay}
    */
-  get emulatedPosition() { return this[PRIVATE].inputSourceImpl.emulatedPosition; }
+  get targetRay() { return this[PRIVATE].targetRay; }
 
   /**
-   * @return {Float32Array}
+   * TODO: Non-standard. Make this read-only in the future.
+   * @param {XRRay} value
+   */
+  set targetRay(value) { this[PRIVATE].targetRay = value; }
```

Comment:
In the future, we can use the exposed `PRIVATE` symbols from #29 to set the targetRay in the GamepadXRInputSource component -- there's a few other spots where the public, exposed API differs from spec, and that change will allow us some greater access internally for these sorts of things -- mostly a comment to myself :smile: